src/cryptography/hazmat/backends/openssl/rsa.py

```diff
@@ -559,6 +565,62 @@ def private_numbers(self):
             )
         )
 
+    def as_bytes(self, encoding, fmt, encryption_algorithm):
```

Comment:
`fmt` makes me sad here, it looks ugly when you pass a kwarg. The objection to `format` is that it's a builtin I assume?